### PR TITLE
added new web-3 onboard

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/common/stores/applications/SsvWeb/Wallet.store.ts [hunk 1]
```diff
@@ -209,7 +270,7 @@ class WalletStore extends BaseStore implements Wallet {
     const myAccountStore: MyAccountStore = this.getStore('MyAccount');
     const applicationStore: Application = this.getStore('Application');
     this.accountAddress = '';
-    this.onboardSdk.walletReset();
+    // this.onboardSdk.walletReset();
```
Comment (meshin-dev) [inline src/app/common/stores/applications/SsvWeb/Wallet.store.ts:273]: Why it was needed wallet reset before and now doesn't?

**Review by meshin-dev — APPROVED** [review]

File: src/app/common/stores/applications/SsvWeb/Wallet.store.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Wallet.store.ts:273]: this function was used when We disconnect from our wallet but in new onboard we don't have this function and also don't need it

File: src/app/common/stores/applications/SsvWeb/Wallet.store.ts [hunk 2]
```diff
@@ -79,31 +83,81 @@ class WalletStore extends BaseStore implements Wallet {
...
-    };
-    console.debug('OnBoard SDK Config:', connectionConfig);
+      accountCenter: {
+        mobile: {
+          enabled: false,
+        },
+        desktop: {
+          position: 'topRight',
+          enabled: false,
+          minimal: false,
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Wallet.store.ts]: done

File: src/app/App.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ const App = () => {
   useEffect(() => {
     if (!applicationStore.locationRestrictionEnabled) {
       console.debug('Skipping location restriction functionality in this app.');
-      walletStore.connectWalletFromCache();
+      walletStore?.checkConnectedWallet();
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/App.tsx]: done

File: public/index.html [hunk 4]
```diff
@@ -33,7 +33,14 @@
 
   <meta
     content="connect-src 'self' data: https://www.gasnow.org
+                        https://rsms.me/inter/font-files/*
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply public/index.html]: done

File: public/images/ssvIcons/logo.svg
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply public/images/ssvIcons/logo.svg:1]: done

**Pull request merged**